src/main/scala/wdlTools/util/Util.scala

```diff
@@ -46,7 +46,18 @@ object URL {
 case class Options(localDirectories: Seq[Path] = Vector.empty,
                    followImports: Boolean = false,
                    verbosity: Verbosity = Normal,
-                   antlr4Trace: Boolean = false)
+                   antlr4Trace: Boolean = false) {
+
+  /**
+    * Clone this Options object, possibly updating attributes.
+    */
+  def clone(localDirectories: Seq[Path] = localDirectories,
+            followImports: Boolean = followImports,
+            verbosity: Verbosity = verbosity,
+            antlr4Trace: Boolean = antlr4Trace): Options = {
+    Options(localDirectories, followImports, verbosity, antlr4Trace)
+  }
+}
 
```

Comment:
If you want to copy an object `x` you can do `x.copy`.  I have used this to modify case classes in a function way: 

val o = Options(...)
val o2 = o.copy(verbosity = ..)

For example: https://github.com/dnanexus/dxWDL/blob/3f3f4b92e59a97863883c2edf1d2b6f27a22362c/src/main/scala/dxWDL/dx/DxFile.scala#L138

Reply:
oh nice, thanks! I'll fix this in the next PR.